src/components/Card/Activity.js

```diff
@@ -4,11 +4,14 @@ import styled from 'styled-components/macro'
 export default function Activity({ activity }) {
   return (
     <ListStyled>
-      <NameStyled>{activity.name}</NameStyled>
-      <SpanStyled>
-        ({activity.hours}{' '}
-        {activity.hours < 1 ? 'min' : activity.hours === 1 ? 'hour' : 'hours'})
-      </SpanStyled>
+      <NameStyled>
+        {activity.name}
+        <SpanStyled>
+          ({activity.hours}{' '}
+          {activity.hours < 1 ? 'min' : activity.hours === 1 ? 'hour' : 'hours'}
```

Comment:
Deine Logik stimmt hier nicht ganz, weil du die Stunden nicht in Minuten übersetzt, wenn es weniger als eine Stunde ist. Wenn man eine Aktivität mit 0.5 speichert, steht da dann `0.5 min`, statt 30 min.
Du müsstest also noch einbauen, dass bei `activity.hours < 1` `activity.hours * 60` genommen wird, bspw. so:
```
<ListStyled>
      <NameStyled>
        {activity.name}
        <SpanStyled>
          ({activity.hours < 1 ? activity.hours * 60 : activity.hours}
          {activity.hours < 1 ? 'min' : activity.hours === 1 ? 'hour' : 'hours'}
          )
        </SpanStyled>
      </NameStyled>
    </ListStyled>
```